skiko/src/jvmMain/cpp/common/awt_jni.cc

```diff
@@ -0,0 +1,67 @@
+#include <cstdint>
+#include <jawt_md.h>
+
+extern "C" jboolean Skiko_GetAWT(JNIEnv *env, JAWT *awt);
+
+extern "C"
+{
+    JNIEXPORT jlong JNICALL Java_org_jetbrains_skiko_AWTKt_getAWT(JNIEnv *env, jobject obj)
+    {
+        JAWT *awt = new JAWT();
+        awt->version = (jint)JAWT_VERSION_9;
+        jboolean result = Skiko_GetAWT(env, awt);
+
+        if (result == JNI_FALSE)
```

Comment:
`if (result == JNI_FALSE) {` I presume.

Reply:
I think we need to preserve the current style (but I like Java-style more :) )

Reply:
Nevermind, we likely need to reformat the whole codebase with clang-format.